tests/assets/dynamic-addon-cards.test.js

```diff
@@ -27,13 +27,17 @@ describe(__filename, () => {
     fetch.mockRestore();
 
     // Inject mozCompare implementation.
-    global.mozCompare = mozCompare;
+    window.mozCompare = mozCompare;
+    // Inject a fake amoTracking object.
+    window.amoTracking = { sendEvent: jest.fn() };
 
     document.body.innerHTML = `<div>${staticCard}</div>`;
 
     return document.querySelector('.StaticAddonCard');
   };
 
+  const flushPromises = () => new Promise((resolve) => setImmediate(resolve));
```

Comment:
Neat, I've not seen this pattern before. Might be worth a comment to explain that it will block until pending promises are resolved.